keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -439,6 +443,42 @@ _Figure: Topology Manager fetches affinity from hint providers._
 * User feedback.
 * *TBD*
 
+# Test Plan
+
+There is a presubmit job for Topology Manager, that will be run on 
+all PRs. This job is here:
+https://testgrid.k8s.io/wg-resource-management#pr-kubelet-serial-gce-e2e-topology-manager
+
+There is a CI job for Topology Manager that will run periodically. This
+job is here:
+https://testgrid.k8s.io/wg-resource-management#pr-kubelet-serial-gce-e2e-topology-manager
+
+The Topology Manager E2E test will enable the Topology Manager
+feature gate and set the CPU Manager policy to 'static'.
+
+At the beginning of the test, the code will determine if the system
+under test has support for single or multi-NUMA nodes. 
+
+## Single NUMA Systems Tests
+For each of the four topology manager policies, the tests will
+run a subset of the current CPU Manager tests. This includes spinning 
+up non-guaranteed pods, guaranteed pods, and multiple guaranteed and 
+non-guaranteed pods. As with the CPU Manager tests, CPU assignment is 
+validated. Tests related to multi-NUMA systems will be skipped, and 
+a log will be generated indicating such.
```

Comment:
With this in place, is there any need for the standalone CPUManager E2E tests, or do these subsume them?

Reply:
Yes, the standalone CPU Manager E2E tests do at least 1 additional test, related to the reconcile cleanup test. That was the test that was failing, and you fixed it with when you cleaned up the reconcile logic. That test is not included in subset that is in Topology Manager E2E tests.

Also, we are looking at extending the CPU Manager E2E tests.